### PR TITLE
Modify candidate table help of yaml format

### PR DESCRIPTION
**Pull request by SaranchukVadym**

### CODE REVIEW

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 databaseChangeLog:
   - changeSet:
-      id: add-data-to-candidate-table
+      id: run-fix-table-candidate
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml:3]: We can't modify existing changesets we should create a new one

File: sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            newColumnName: CN_OTHER_SKILLS
+            oldColumnName: CN_EXPERIENCE
+            defaultValue: NULL
+            tableName: CANDIDATE
+
+  - changeSet:
+      id: 24.04.2021_rename_CN_EXPERTISE_column_in_candidate_table
+      author: Saranchuk Vadym
+      preConditions:
+        onFail: CONTINUE
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml:24]: On fail should be MARK_RAN

File: sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml [hunk 3]
```diff
@@ -0,0 +1,124 @@
+databaseChangeLog:
+  - changeSet:
+      id: 24.04.2021_rename_CN_EXPERIENCE_column_in_candidate_table
+      author: Saranchuk Vadym
+      preConditions:
+        onFail: CONTINUE
+        tableExists:
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml:7]: columnExists precondition is enough here, tableExists not need

File: sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+            newColumnName: CN_MAIN_SKILL
+            oldColumnName: CN_EXPERTISE
+            tableName: CANDIDATE
+
+  - changeSet:
+      id: 24.04.2021_drop_CN_LOCATION_column_in_candidate_table
+      author: Saranchuk Vadym
+      preConditions:
+        onFail: CONTINUE
+        tableExists:
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml:42]: the same

**Review by ikantsevich — CHANGES_REQUESTED** [review]

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml (hunk 1 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml:3]: corrected

File: sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml (hunk 3 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml:7]: corrected

File: sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml (hunk 4 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml:42]: corrected

File: sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml (hunk 2 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/24.04.2021_modify_candidate_table.yaml]: corrected

**Review by ikantsevich — APPROVED** [review]

**Pull request merged**